src/main/java/seedu/todo/commons/util/TimeUtil.java

```diff
@@ -248,6 +249,16 @@ public boolean isOngoing(LocalDateTime startTime, LocalDateTime endTime) {
         
         return LocalDateTime.now(clock).isAfter(startTime) && LocalDateTime.now(clock).isBefore(endTime);
     }
+    
+    public boolean isToday(ImmutableTask task){
+        if (!task.getStartTime().isPresent() && !task.getEndTime().isPresent()){
```

Comment:
No double negation with `&&` please, this is very hard to read. Write `boolean isContainsTime = x.isPresent() || y.isPresent()` then `if (!isContainsTime )...` or something